learn/intro.md

```diff
@@ -49,6 +49,7 @@ If you're familiar with Rollkit, you may want to skip to the [tutorials section]
 
 - [Wordle app](/tutorials/wordle)
 - [CosmWasm rollup](/tutorials/cosmwasm)
+- [zkML rollup](/tutorials/zkml.md)
```

Comment:
@Manav-Aggarwal I think we should link to the tutorials page on intro.md, but maybe not duplicate everything that is also in the tutorials/overview.md page .what do you think? in short, I think we only need to list them out once in the site, for maintenance reasons (as this page intro.md is outdated and doesn't contain polaris, etc.)

Reply:
Yes agreed, made a PR here: https://github.com/rollkit/docs/pull/337